src/app.js

```diff
@@ -276,11 +276,16 @@ export function app(app) {
           removeElement(element, reusableChild[0], reusableNode)
         }
       }
-    } else if (node !== oldNode) {
+    } else if (
+      element != null &&
+      node !== oldNode &&
+      node !== element.nodeValue
+    ) {
       var i = element
       parent.replaceChild((element = createElement(node)), i)
```

Comment:
@ngryman It would avoid creating a new text node when both node and oldNode are textnodes and only the text content has changed. Why `nodeValue` over `textContent`?